### PR TITLE
Adds translated string for user kind in the user table

### LINKED ISSUE #9600: Use translated string for user kind in the user table
## Observed behavior

In the user table component, we render visually hidden user kind for assistive technologies:

https://github.com/learningequality/kolibri/blob/82ae0b3997943d6d349080b0df80c10c8bb8c1fe/kolibri/plugins/facility/assets/src/views/UserTable.vue#L89-L91 

It is supposed to convey the same information as `aria-hidden` badge (e.g. _Super admin_):

Screenshot from 2022-08-11 13-04-47

https://github.com/learningequality/kolibri/blob/82ae0b3997943d6d349080b0df80c10c8bb8c1fe/kolibri/plugins/facility/assets/src/views/UserTable.vue#L78-L87

However, the visually hidden `<td>` only shows an untranslated constant (e.g. _superuser_):

Screenshot from 2022-08-11 13-13-07

## Expected behavior

Instead of rendering `user.kind` constant, we need to render translation corresponding to it (the same one as the text shown in badges)

## Comments

There is already a translation map implemented in `UserTypeDisplay`

https://github.com/learningequality/kolibri/blob/82ae0b3997943d6d349080b0df80c10c8bb8c1fe/kolibri/core/assets/src/views/UserTypeDisplay.vue#L39-L48

Ideally, it'd be extracted to common utilities or constants and then imported to all places that need to use it.

## User-facing consequences

Worse user experience for people using screenreaders who will hear raw constant value instead of user-friendly and properly translated text.

## Steps to reproduce

This can be seen at more places where `UserTable` is used. One of them is the facility users page. To see the table, sign-in as admin and go to _/facility/#/users_. Preview markup.

## Context

- Kolibri version: Kolibri 0.16.0.dev0+git.20220811104159
- Operating system: Ubuntu 20.04.3 LTS
- Browser: Chrome Version 104.0.5112.79 (Official Build) (64-bit)

### PR DESCRIPTION
**Pull request by akolson**
## Summary

In the users table under _Facility_, we render a visually hidden `<td>` with a user kind value for assistive technologies. However, it shows an untranslated constant (e.g. superuser). This PR resolves this issue by rendering the translated, constant equivalent string into the  visually hidden `<td>` (e.g. Super admin), the same as the one displayed in the badge.

## References

Fixes https://github.com/learningequality/kolibri/issues/9600

## Reviewer guidance

- Navigate to _Facility_ menu option, and select the _Users_ tab ie. _facility/#/users_
- Check any of the rows markup in the Users table.
- There should be a `<td>` with a class `visuallyHidden`, containing the same [translated] string as that displayed in the badge for that particular row.

----

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/assets/src/views/UserTypeDisplay.vue
Comment (MisRob) [inline kolibri/core/assets/src/views/UserTypeDisplay.vue:35]: It may be helpful to briefly comment that the string needs to stay here because we use `UserTypeDisplay` from the `crossComponentTranslator`

**Review by MisRob — APPROVED** [review]
Thank you @akolson. The code is looking good to me. @radinamatic as it's an a11y update, would you like to test this out before we merge it?

**Review by MisRob — DISMISSED** [review]
Blocking merging until 0.15.7 "Planned Patch 5" is tagged on `release-v0.15.x` as this PR targets the next "Planned Patch 6"

File: kolibri/core/assets/src/views/UserTypeDisplay.vue
Comment (akolson) [reply kolibri/core/assets/src/views/UserTypeDisplay.vue:35]: Sure, let me do that

**Review by rtibbles — APPROVED** [review]
Reaffirming @MisRob's approval!

**Pull request merged**
